api/managementpb/backup/backups.proto

```diff
@@ -177,6 +177,27 @@ message RemoveScheduledBackupRequest {
 
 message RemoveScheduledBackupResponse {}
 
+message GetLogsRequest {
+  string artifact_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  uint32 from_chunk = 2;
+  uint32 limit = 3;
+}
+
+message GetLogsResponse {
+  repeated LogChunk logs = 1;
+}
+
+// LogChunk represent one chunk of logs.
+message LogChunk {
+  uint32 chunk_id = 1;
+  string message = 2;
+  bool last_chunk = 3;
```

Comment:
how you will detect that returned log chunk is the last one? Maybe better to move this field to just GetLogsResponse?

Reply:
pmm-agent, after streaming logs is finished, will send special message that logs are finished and pmm-managed will set this flag for last chunk

Reply:
> @Dasio Wouldn't it be helpful to add a timestamp to the log chunk?

What if we don't have timestamp available? (eg. in xtrabackup-mysql). In MongoDB it's inside message so we have this information.

Reply:
`hasNext` but what if we don't know yet it has next or not. We just know that it maybe it's not end. Then after some time (pmm-agent sends done message), there will be no logs. (In pmm-managed I set flag for last chunk)

Reply:
added `end` as it is inside GetLogsResponse